Source/camera/camera.cpp

```diff
@@ -17,7 +18,7 @@ void Camera::perFrameTimeLogic() {
 }
 
 glm::mat4 Camera::getMat4Projection() {
-	glm::mat4 projection = glm::perspective(glm::radians(FOV), (float)SCR_WIDTH / (float)SCR_HEIGHT, 0.1f, 100.0f);
+	glm::mat4 projection = glm::perspective(glm::radians(FOV), (float)context::window::getWidth() / (float)context::window::getHeight(), 0.1f, 100.0f);
```

Comment:
This ratio will be useful elsewhere too, might want to consider a `context::aspectRatio()` function.